### PR TITLE
Updates to clarify WG and Donation guidelines

### PR DESCRIPTION
**Pull request by paulgrehan**
Per discussions relating to WG and Donation clarity, updates to WG Lifecycle and Donations guidelines reflect.

### CODE REVIEW

**Comment by paulgrehan** [comment]
@bumblefudge - please apply your literary panache

**Review by bumblefudge — APPROVED** [review]
LGTM but maybe an additional review from TSC and/or ED would help

File: work-item-lifecycle.md [hunk 1]
```diff
@@ -30,6 +30,15 @@ Group chair will designate each submission as a “Pre-Draft” document.
 2. Additional “Straw men” can serve as input documents to a Draft Deliverable,
    in part or in whole, and can be submitted after a Draft has been initiated at
    the discretion of the chairs.
+3. If a deliverable builds on prior art designed and/or built by a participant
+   in the DIF work in a legally significant way, i.e., if one or more input
+   documents to the DIF work could constitute a threat to the IP safety of the
+   DIF work item, the Working Group MAY request that it be officially donated to
```
Comment (brentzundel) [inline work-item-lifecycle.md:36]:
```suggestion
   DIF work item, the Working Group SHOULD request that it be officially donated to
```

File: working-group-lifecycle.md [hunk 2]
```diff
@@ -8,26 +8,99 @@ The DIF Working Group process is set up as follows:
...
+2.**Writing the WG charter**. Prior to setting up the WG charter, DIF members
+and interested parties (which may need to join DIF or a co-sponsoring liasion
+organization before signing) are advised to host non-IPR protected meetings to
+finalize the WG Charter. While it can be challenging to do so in a
+non-IPR-protected context, it is essential that the group define a scope for the
+group and itemize both the "in-scope" and "out-of-scope" sections of the charter
+first. To avoid duplication of effort, it is advised to approach existing WG's
+to confirm there is no overlap or conflict between the scopes of bith charters.
```
Comment (brentzundel) [inline working-group-lifecycle.md:37]:
```suggestion
to confirm there is no overlap or conflict between the scopes of both charters.
```

**Review by brentzundel — APPROVED** [review]

**Comment by csuwildcat** [comment]
+1

**Review by peacekeeper — APPROVED** [review]

**Comment by andorsk** [comment]
Any reason this isn't merged. Suggest merging now

**Review by kimdhamilton — APPROVED** [review]
This is a good contribution:
- Donation guidelines clarifies the distinction between donating to DIF vs a WG accepting the work item, as recently discussed
- Other than that, formatting and typo fixes

**Pull request merged**